src/tls/connection.c

```diff
@@ -371,12 +371,10 @@ request_dynamic_wsinstance (const Fingerprint *fingerprint)
       goto out;
     }
 
-  sockaddr_printf (&addr, "%s/https-factory.sock", parameters.wsinstance_sockdir);
-
-  debug (CONNECTION, "  -> connecting to %s...", addr.sun_path);
-  if (connect (fd, (struct sockaddr *) &addr, sizeof addr) != 0)
+  debug (CONNECTION, "  -> connecting to https-factory.sock");
+  if (af_unix_connectat (fd, parameters.wsinstance_sockdir, "https-factory.sock") != 0)
     {
-      warn ("connect(%s) failed", addr.sun_path);
+      warn ("connect failed");
```

Comment:
Please still let this say "https-factory.sock", so that in case of failure it's easier to debug (the directory is obvious from that).

Reply:
Done.